tools/aa/aa_tool.sh

```diff
@@ -71,9 +71,9 @@ function aa {
   STARTUP_OS=$AA_BASE/head/startup-os
 
   # Uncomment to override StartupOS repo:
-  #STARTUP_OS=<repo path>
+  STARTUP_OS=/home/oferb/devel/base/ws/aa_checks2/startup-os
```

Comment:
@oferb I know it's a draft, but you utilize something like [direnv](https://direnv.net/) to switch envs as you switch directories when testing locally